Documentation/contributing/development/statedb.rst

```diff
@@ -398,12 +410,51 @@ a struct as this is the prevalent style in Cilium.
 
 For a real-world example see ``pkg/maps/bwmap/cell.go``.
 
+
+Script commands
+~~~~~~~~~~~~~~~
+
+StateDB comes with a rich set of script commands for inspecting and manipulating tables:
+
+.. code-block:: shell-session
+  :caption: example.txtar
+
+  # Show the registered tables
+  db
+
+  # Insert an object
+  db/insert my-table example.yaml
+
+  # Compare the contents of 'my-table' with a file. Retries until matches.
+  db/cmp my-table expected.table
+
+  # Show the contents of the table
+  db/show
+
+  # Write the object to a file
+  db/get my-table 'Foo' --format=yaml --out=foo.yaml
+
+  # Delete the object and assert that table is empty.
+  db/delete my-table example.yaml
+  db/empty my-table
+
+  -- expected.table --
+  Name  Color
+  Foo   Red
+
+  -- example.yaml --
+  name: Foo
+  color: Red
+
+See ``help db`` for full reference in ``cilium-dbg shell`` or in the ``break`` prompt in tests.
+A good reference is also the existing tests. These can be found with ``git grep db/insert``.
+
 Metrics
 ~~~~~~~
 
 Metrics are available for both StateDB and the reconciler, but they are disabled
 by default due to their fine granularity. These are defined in ``pkg/hive/statedb_metrics.go``
-and ``pkg/hive/reconciler_metrics.go``. As this documentation is manually maintained, it may
+and ``pkg/hive/reconciler_metrics.go``. As this documentation is maintained by hand, it may
```

Comment:
nit: "manually maintained" seems a bit more clear for international readers than using the "by hand" expression.

Reply:
Makes sense. Reverted.